### DISCUSSION TITLE
About SRS Users section

### DISCUSSION THREAD

**Discussion by imranertaza** [edrisranjbar/qurandl#34] [General]
Hi
May I know why do we need user registration? As anybody from anywhere can search recitations for recitations by surah or reciter. So why a normal user should register here?

And what will be there inside user panel after sign in? please let me know.
Thank you

**Comment by edrisranjbar** [comment]
almost all of the features we provide in the website does not require logging in, but for the near future we need to have profile for each user, so we can track, store and analyze personal data and suggest best recitations based on most listened and searched `Qaris` and also adding recitations to `wishlist` and stuff like that.
